Make HM constants dependency consistent to avoid duplicates.

dbatiste opened the pull request.

m6jones commented:
You are running into the Chester's new lint configuration. He fixed this repo so it is sort of odd.

dbatiste commented:
I think this is because we updated the sort config to have special sort rules for static properties and static styles, but we updated it after his fixes.  I'll make a fix for the ordering, although perhaps we need another special rule for `is`, since we probably want that first.
